editor/plugins/canvas_item_editor_plugin.cpp

```diff
@@ -2599,6 +2599,9 @@ void CanvasItemEditor::_gui_input_viewport(const Ref<InputEvent> &p_event) {
 
 void CanvasItemEditor::_update_cursor() {
 	CursorShape c = CURSOR_ARROW;
+	bool should_switch = false;
+	if (drag_selection.size() != 0)
+		should_switch = abs(fmod(drag_selection[0]->_edit_get_rotation(), M_PI)) >= M_PI_4 && abs(fmod(drag_selection[0]->_edit_get_rotation(), M_PI)) <= M_PI - M_PI_4;
```

Comment:
If you replace `M_PI` with `Math_PI`, the CI should build without any issues. For `M_PI_4`, you'll need to use something like `Math_PI / 4.0`. I mentioned this on the IRC, but was too slow in my reply and I do not use Discord, so I thought I'd mention it here.

Also: You will likely need to squish your commits. There is a page on the [documentation](https://docs.godotengine.org/en/stable/community/contributing/pr_workflow.html?highlight=rebase#the-interactive-rebase) that helps explain how to do this.

Outside of those two minor things, the changes looks good to me! :+1:

Reply:
Alright, thanks! Will do!

Reply:
Done!